xml/System.Net.Http/HttpClient.xml

```diff
@@ -1613,6 +1668,9 @@ The <paramref name="requestUri" /> is not an absolute URI.
 
 ## Remarks
  This operation doesn't block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response body is read.
+ Internally, this operation uses <xref:System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode%2A?displayProperty=nameWithType> to ensure the response status code is in the success range (200-299) and throws an <xref:System.Net.Http.HttpRequestException> if it is not.
```

Comment:
Same comment as I left in the other file.

Reply:
Apologies, I couldn't figure out what comment do you refer to. What would be your suggestion for this line?